v3/src/components/case-card/case-card-header.tsx

```diff
@@ -1,18 +1,80 @@
 import { observer } from "mobx-react-lite"
 import React from "react"
-import { CollectionTitle } from "../case-tile-common/collection-title"
-import { useCaseCardModel } from "./use-case-card-model"
 import { useCollectionContext } from "../../hooks/use-collection-context"
+import { IDataSet } from "../../models/data/data-set"
 import { createCasesNotification } from "../../models/data/data-set-notifications"
-import { t } from "../../utilities/translation/translate"
 import { IGroupedCase } from "../../models/data/data-set-types"
 import { setSelectedCases } from "../../models/data/data-set-utils"
+import { t } from "../../utilities/translation/translate"
+import { CollectionTitle } from "../case-tile-common/collection-title"
+import { useCaseCardModel } from "./use-case-card-model"
 
 import Arrow from "../../assets/icons/arrow.svg"
 import AddIcon from "../../assets/icons/add-data-icon.svg"
 
 import "./case-view.scss"
 
+interface ICaseCardControlsProps {
+  caseIndexText: string
+  cases: IGroupedCase[]
+  data?: IDataSet
+  displayedCaseIndex: number
+  isCollectionSummarized: boolean
+}
+
+function CaseCardControls({
```

Comment:
`CaseCardControls` could warrant its own file at this point, but I'm also fine with leaving it here if you prefer.